### ISSUE TITLE
'product = atmos.25' not working for GEFS

### ISSUE THREAD

**Issue by Michael-Barletta** [blaylockbk/Herbie#222]
When trying to load in the quarter degree GEFS, no matter the date, I am getting an error saying Herbie did not find it. All other products specified in the Herbie documentation seem to be working fine with the GEFS however.

`H = Herbie(
    "2023-07-29 00:00",
    model="gefs",
    product="atmos.25",
    member="mean",
)`
💔 Did not find ┊ model=gefs ┊ product=atmos.25 ┊ 2023-Jul-29 00:00 UTC F00

**Comment by blaylockbk** [comment]
Since you closed this, did you find a resolution to your issue? How did you resolve it?

**Comment by Michael-Barletta** [comment]
Hi,

I never did find a solution, but I closed the issue because I believe the grib files are no longer available on AWS for the GEFS 0.25, 0.5 still works though, I could be wrong. I emailed your utah email asking a few herbie questions but maybe they're better off asked here. I am running into the same issue with the RRFS but I do see those files available on AWS and don't want to open another issue on github unless I have to.

It also seems that the RRFS data is not available through herbie either for any dates close to the current day although I see it available on AWS.

H = Herbie(
    "2023-01-04 12:00",
    model="rrfs",
    fxx=1,
    member=1,
)

Thanks,
-Michael



-----------------------------------------------
Michael N. Barletta
***@***.***

SUNY Albany
Atmospheric Science
BS. 23', MS. 25'

(516).557.3434

________________________________

**Comment by johnrobertlawson** [comment]
I'm also unable to get the 0.25-deg GEFS data, so I went into the Herbie source code to test the file-path string creation. It still generates a valid AWS link; for instance, comparing 

``gefs.20231104/00/atmos/pgrb2sp25/gec00.t00z.pgrb2s.0p25.f006`` 

to [a recent AWS folder](https://noaa-gefs-pds.s3.amazonaws.com/index.html#gefs.20231104/00/atmos/pgrb2sp25/). I don't think I saw any URL differences.
